subprojects/testing-jvm/src/main/java/org/gradle/api/internal/tasks/testing/junitplatform/JUnitPlatformSpec.java

```diff
@@ -28,13 +27,20 @@ public class JUnitPlatformSpec extends AbstractJUnitSpec {
     private final Set<String> includeTags;
     private final Set<String> excludeTags;
 
-    public JUnitPlatformSpec(JUnitPlatformOptions options, Set<String> includedTests,
-                             Set<String> excludedTests, Set<String> includedTestsCommandLine) {
+    public JUnitPlatformSpec(
```

Comment:
This constructor is a logical error timebomb.

Reply:
See response below